src/blockchain/node/p2p/Requestor.cpp

```diff
@@ -602,34 +560,38 @@ auto Requestor::Imp::update_remote_position(
     const network::p2p::State& state) noexcept -> void
 {
     remote_position_ = state.Position();
-    update_activity();
     last_remote_position_ = Clock::now();
 }
 
-auto Requestor::Imp::work() noexcept -> bool
+auto Requestor::Imp::work() noexcept -> int
 {
+    tdiag(
+        "RRRR work(), state = ",
```

Comment:
are those "RRRR" logs just temporary?:D if not maybe you can come up with better word:D

Reply:
Any tdiag call with this a tag formed by an upper case letter repeated 3 or 4 times is temporary, the tag makes it easy to do a issue-specific grep. At some stage, they should be removed.

Reply:
ok, good